recipe/session/logger_test.go

```diff
@@ -0,0 +1,166 @@
+package session
+
+import (
+	"bytes"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/supertokens/supertokens-golang/supertokens"
+	"github.com/supertokens/supertokens-golang/test/unittesting"
+)
+
+// Added the logger tests here because supertokens/logger_test.go causes cyclic import errors due to imports in test/unittesting/testingUtils.go
+
+func resetLogger() {
+	supertokens.Logger = log.New(os.Stdout, "com.supertokens", 0)
+	os.Unsetenv("SUPERTOKENS_DEBUG")
```

Comment:
shouldnt debugEnabled be reset to false after the test?

Reply:
Done